scripts/test-upgrade.sh

```diff
@@ -130,3 +130,5 @@ bash scripts/deploy-rpc-playbooks.sh
 #              the rpc_post_upgrade role directory.
 cd ${RPCD_DIR}/playbooks
 openstack-ansible rpc-post-upgrades.yml
+openstack-ansible -e 'logging_upgrade=true' logstash.yml
```

Comment:
rpc-post-upgrades.yml is essentially a verification step, maybe it would be better to add this to task 7 or between 7 and 8.